### PR TITLE
vertical tabs component and design

### PR DESCRIPTION
**Pull request by zivl**
the selection is not yet completed.
I'll work it on the next PR

### CODE REVIEW

File: src/VerticalTabsItem/VerticalTabsItem.js [hunk 1]
```diff
@@ -64,7 +58,8 @@ class VerticalTabsItem extends React.PureComponent {
   }
 
   _renderPrefix() {
-    const { prefixIcon, size, type } = this.props;
+    const { prefixIcon, type } = this.props;
+    const { size } = this.context;
```
Comment (mosheKerbel) [inline src/VerticalTabsItem/VerticalTabsItem.js]: Seems like you are using `context` in a deprecated way
image

deprecated log referecned here - https://reactjs.org/docs/legacy-context.html

File: src/VerticalTabsItem/VerticalTabsItem.js [hunk 2]
```diff
@@ -42,12 +37,11 @@ class VerticalTabsItem extends React.PureComponent {
...
   static defaultProps = {
     type: 'tab',
-    size: 'medium',
-    selected: false,
   };
 
   _renderText() {
-    const { children, size, type } = this.props;
+    const { children, type } = this.props;
+    const { size } = this.context;
```
Comment (mosheKerbel) [inline src/VerticalTabsItem/VerticalTabsItem.js]: title should also be affected by the small/medium size
image

File: src/VerticalTabsItem/VerticalTabsItem.js (hunk 1 above)
Comment (zivl) [reply src/VerticalTabsItem/VerticalTabsItem.js]: I disagree: https://reactjs.org/docs/context.html#classcontexttype

File: src/VerticalTabsItem/VerticalTabsItem.js (hunk 2 above)
Comment (zivl) [reply src/VerticalTabsItem/VerticalTabsItem.js]: no. title is always in "small" size. check the zeplin

File: src/VerticalTabsItem/VerticalTabsItem.js (hunk 1 above)
Comment (mosheKerbel) [reply src/VerticalTabsItem/VerticalTabsItem.js]: yes, I see
I don't know why my Webstorm is yelling about it.

File: src/VerticalTabsItem/VerticalTabsItem.js (hunk 2 above)
Comment (mosheKerbel) [reply src/VerticalTabsItem/VerticalTabsItem.js]: Yes, I see now the IC Zepplin.
The other Zepplin of the [bigger component](https://app.zeplin.io/project/5c8f6f1f9f489505cac756dc/screen/5d3030d7a4f29f5171bb9dce)  was not aligned.

**Pull request merged**